templates/_index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<meta http-equiv="X-UA-Compatible" content="ie=edge">
+	<title>Webpack PWA App</title>

```

Comment:
This should be the application name right?

Reply:
Yes, But for that, we have to make app name to be global but that for now local to `prompting()`.
If we make manifest details global we can also add `meta description` to our app. Should I go ahead for this change @sendilkumarn?